--- python/python3/CMakeLists.txt
+++ python/python3/CMakeLists.txt
@@ -34,7 +34,7 @@ if ( NOT DEFINED ENABLE_SWIG_PYTHON3 OR ENABLE_SWIG_PYTHON3 )
           ${PROJECT_SOURCE_DIR}/common/module
           ${PROJECT_BINARY_DIR}/common/module
           ${PROJECT_BINARY_DIR}/src/client
-          ${nds2-client_INCLUDE_DIRS}
+          ${NDS_CLIENT_INCLUDE_DIRS}
           ${PYTHON3_NUMPY_INCLUDE_PATH}
           ${PYTHON3_INCLUDE_PATH}
           )
@@ -51,7 +51,7 @@ if ( NOT DEFINED ENABLE_SWIG_PYTHON3 OR ENABLE_SWIG_PYTHON3 )
           PREFIX         PYTHON3
           INTERFACE_FILE ${INTERFACE_FILE}
           MODULE         nds2
-          LINK_LIBRARIES nds::cxx
+          LINK_LIBRARIES ${ndscxx_LIBRARY_PATH}
           SWIG_FLAGS "-py3" "-shadow" "-importall" "-Wall" "-threads" "-O"
           )
       set( NDS_PYTHON_VERSION_MAJOR ${PYTHON3_VERSION_MAJOR} )
